pkg/monitor/monitor/util.go

```diff
@@ -458,6 +465,9 @@ func getMonitorPrometheusContainer(monitor *v1alpha1.TidbMonitor, tc *v1alpha1.T
 	if monitor.Spec.Prometheus.Config != nil && len(monitor.Spec.Prometheus.Config.CommandOptions) > 0 {
 		c.Command = append(c.Command, monitor.Spec.Prometheus.Config.CommandOptions...)
 	}
+	if monitor.Spec.Thanos != nil {
+		c.Command = append(c.Command, "--storage.tsdb.max-block-duration=2h")
```

Comment:
Do we need to set `--storage.tsdb.min-block-duration` here too?

Reply:
I think we need.

Reply:
When we use thanos ,we need to disable prometheus compact.The --storage.tsdb.min-block-duration and --storage.tsdb.max-block-duration must be set to equal values to disable local compaction .

Reply:
https://thanos.io/v0.11/components/sidecar/